app/views/tanks/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<section class="section facilities__index">
+  <div class="container facilities__index__container">
+    <p id="notice"><%= notice %></p>
```

Comment:
Im just now "noticing" 😏 That we render `notice` both here and in the application layout [here](https://github.com/rubyforgood/abalone/blob/master/app/views/layouts/application.html.erb#L102-L107). We should make a follow up issue to unify these so that its only need in one place, and probably use `flashes` instead of `notice` and `alert`. 

Not an action item here, but wanted to "note" it so it didnt get forgotten.